Microsoft.Toolkit.Graph.Controls/Providers/InteractiveProviderBehavior.cs

```diff
@@ -2,10 +2,21 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if DOTNET
+using System;
+using System.Linq;
+using System.Windows;
+using Microsoft.Toolkit.Graph.Providers;
```

Comment:
@azchohfi I realize I have the same namespace in the .NET Standard project for the base providers as I do within the other `Microsoft.Toolkit.Graph.Controls` package for the XAML based provider behaviors.

Should I change the UWP namespace to be like `Microsoft.Toolkit.Graph.Behaviors` or something for the XAML based ones or it alright if they overlap (assuming no classes have the same name)?